packages/mds-repository/connection.ts

```diff
@@ -84,23 +86,12 @@ export const ConnectionManager = (prefix: string, options: Omit<ConnectionManage
     return connections[mode]
   }
 
-  const connect = async (mode: ConnectionMode) => {
-    const connection = await getConnection(mode)
-    if (!connection) {
-      /* istanbul ignore next */
-      throw RepositoryError(Error(`Connection ${connectionName(mode)} not found`))
-    }
-    if (!connection.isConnected) {
-      try {
-        await connection.connect()
-      } catch (error) /* istanbul ignore next */ {
-        throw RepositoryError(error)
-      }
-    }
-    return connection
-  }
+  public initialize = async (): Promise<void> => {
+    const { connect, options } = this
+    const {
+      PG_MIGRATIONS = 'true' // Enable migrations by default
```

Comment:
I'm a fan of this...perhaps another place for envalid (in the future)

Reply:
Or even the `getEnvVar` method we have in `mds-utils`